### ISSUE TITLE
Name Property missing for group info lists present in "What's New" page.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#326]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Navigate to "Recently added samples" or "Recently updated samples" in Whats New page and observe what narrator announces.

**Expected behavior**

There should be a correct group name.

**Actual behavior**

Narrator announces group info list as "App UI basics...".

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72558283-3c1c0180-3857-11ea-926e-b71fdac9ecd2.png)

![image](https://user-images.githubusercontent.com/13142665/72558311-463e0000-3857-11ea-9852-c5b9050ecf20.png)

![image](https://user-images.githubusercontent.com/13142665/72558325-50f89500-3857-11ea-8a3c-9f8f0eb8c636.png)

![image](https://user-images.githubusercontent.com/13142665/72558360-5e158400-3857-11ea-990c-abd49629bcb7.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://dev.azure.com/microsoft/OS/_workitems/edit/24649990)
